### PR TITLE
Exclude empty tags from saving

### LINKED ISSUE #9020: Empty Tag
### Steps to reproduce

1.Press edit tags on any envelope
2. add an empty tag
image

### Expected behavior

Error message or disabled button, maybe also add a check in the controller

### Actual behavior

Tag is created

### Mail app version

main

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by GretaD**
fixes #9020

### CODE REVIEW

**Comment by hamza221** [comment]
I was thinking of just disabling the button if there's no text

File: src/components/TagModal.vue [hunk 1]
```diff
@@ -177,7 +177,11 @@ export default {
 				showError(this.t('mail', 'Tag already exists'))
 				return
 			}
-			if (displayName !== null && displayName !== '') {
+			if (displayName.trim() === '') {
+				showError(this.t('mail', 'Tag name cannot be empty'))
+				return
+			}
+			if (displayName !== '') {
```
Comment (ChristophWurst) [inline src/components/TagModal.vue:184]: isn't this check superfluous if you're testing the trimmed name for emptyness above?

Comment (GretaD) [reply src/components/TagModal.vue:184]: true, i didnt add this one, it was there, but i should have removed it instead of simplifying it.

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**